torch_tvm/operators.cpp

```diff
@@ -23,12 +23,18 @@ std::unordered_map<Symbol, TVMOpFunctor>& getTVMOperatorMap() {
   return map;
 }
 
+std::unordered_map<Symbol, ParamIndicesType>& getOpParamsMap() {
```

Comment:
I wonder if we should change getTVMOperatorMap to just return the `struct TVMOpMap` (and maybe even rename that to something a bit more understandable)

Reply:
It might be better. I had to add `getOpParamsMap` and corresponding `param_map` because we dont save TVMOpMap. Also probably should rename `TVMOpMap` to `TVMOp`?